meson.build

```diff
@@ -733,12 +733,10 @@ if features['sdl2-gamepad']
     sources += files('input/sdl_gamepad.c')
 endif
 
+# mandatory C11 atomics
 stdatomic_dep = cc.find_library('atomic', required: false)
```

Comment:
you can move to the top of the file along with other unconditional dependencies

Reply:
This is technically conditional though. It depends on your compiler, so I guess in that section with dl makes the most sense.